### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by bindrad**

### CODE REVIEW

File: .github/workflows/conf/yamllint.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+extends: default
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - '.yamllint'
+ignore: |
+  vendor
+  shim
+rules:
```
Comment (akshayashu) [inline .github/workflows/conf/yamllint.yaml:10]: Good job

**Review by akshayashu — CHANGES_REQUESTED** [review]

**Comment by akshayashu** [comment]
@Mergifyio refresh

**Comment by bindrad** [comment]
@Mergifyio refresh

**Review by akshayashu — APPROVED** [review]

**Comment by akshayashu** [comment]
@Mergifyio refresh

**Pull request merged**